### PR TITLE
Introduce some navigation steps to about page tour

### PR DESCRIPTION
**Pull request by markryall**
This PR introduces some steps to the About Page tour so that the tour can start on the checklist page.

This makes the tour more resilient in case the about page has a different id and also avoids the issue where the tour would not launch on initial click (due to the sequence of events).

The copy for the steps is a placeholder at this stage.

### CODE REVIEW

**Comment by fditrapani** [comment]
This approach works really nice. I can go in and edit the copy. It unfortunately doesn't work so great on mobile (something we can tackle in another PR).

**Comment by fditrapani** [comment]
This is starting to feel real good. I made some updates to the text and one of the tours:

about

File: client/layout/guided-tours/tours/checklist-about-page-tour.js [hunk 1]
```diff
@@ -26,31 +25,51 @@ export const ChecklistAboutPageTour = makeTour(
...
 						'This is the internet and we can update it as many times as we want. ' +
 						'The key is to just get it started.'
 				) }
 			</p>
 			<p>
 				{ translate(
-					'Let’s start by changing the default text with an introduction. ' +
-						'Here are some questions to help you out: Who are you and where are you based? ' +
-						'Why did you start this site? ' +
+					'Let’s change the default text with a new introduction. Here are some questions ' +
```
Comment (a8ci18n) [inline client/layout/guided-tours/tours/checklist-about-page-tour.js]: Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Let’s start by changing the default text with an introduction. Here are some questions to help you out: Who are you and where are you based? Why did you start this site? What can visitors expect to get out of it?' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/layout/guided-tours/tours/checklist-about-page-tour.js [hunk 2]
```diff
@@ -26,31 +25,51 @@ export const ChecklistAboutPageTour = makeTour(
...
+					components: { b: <strong /> },
+				} ) }
+			</p>
+			<Continue target="page-about" step="about-page" click hidden />
+		</Step>
+
+		<Step name="about-page" placement="right">
+			<p>
+				{ translate(
+					'Welcome to your About Page — this is often the most visited page on a site. ' +
```
Comment (a8ci18n) [inline client/layout/guided-tours/tours/checklist-about-page-tour.js]: Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'The About Page is often the most visited page on a site. You might find that it never feels quite done - that’s OK. This is the internet and we can update it as many times as we want. The key is to just get it started.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by fditrapani** [comment]
LGTM 👍

File: client/layout/guided-tours/tours/checklist-about-page-tour.js [hunk 3]
```diff
@@ -26,31 +25,46 @@ export const ChecklistAboutPageTour = makeTour(
...
+			placement="beside"
+			arrow="left-top"
+			target="side-menu-page"
 			style={ {
-				animationDelay: '0.7s',
+				animationDelay: '0s',
 			} }
 		>
+			<p>
+				{ translate( 'Click on {{b}}Site Pages{{/b}} to see all the pages on our site.', {
```
Comment (yoavf) [inline client/layout/guided-tours/tours/checklist-about-page-tour.js]: @markryall this should probably be `your site` instead of `our site`, right?

**Pull request merged**